docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -262,6 +267,53 @@ docker run -i --rm -p 8080:8080 quarkus-quickstart/getting-started
 
 NOTE: Interested by tiny Docker images, check the {quarkus-images-url}/graalvm-{graalvm-version}/distroless[distroless] version.
 
+== Producing a container with a multi-stage Docker build
```

Comment:
I would prefer we said `Creating` a container instead of `Producing`, since that is the industry standard

Reply:
The previous paragraph title is "Producing a container", so if I rephrase this one I should also rephrase the previous one ?

Reply:
Yes please

Reply:
done